src/components/PostDetails/PostDetails.scss

```
@@ -7,6 +7,10 @@
     text-align: right;
   }
 
+  &__visiblelist {
+    display: none;
+  }
```

in case of this class name, I'm waiting that component would be a visible. 
Visible or unvisible state is BEM modifier and should be written with '--' prefix
```suggestion
  &__list--unvisible {
    display: none;
  }
```